sentences/cs/homeassistant_HassTurnOff.yaml

```diff
@@ -2,4 +2,6 @@ language: cs
 intents:
   HassTurnOff:
     data:
-    - sentences: []
+      - sentences: 
+          - "<vypnout> <name>"
```

Comment:
Name is not expansion rule, therefore I think the tests will fail here.
`<vypnout> {name}` should work.